recipes/openassetio/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+  "1.0.0-alpha.6":
```

[fix] Should this be:
```suggestion
  "1.0.0-alpha.7":
```

---

Oh dear, that's worrying, definitely updated that... checking.

---

Nope genuinely missed that one, good spot, fixed in f92d01e8